src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1354,7 +1354,7 @@ bool primitive_inst::has_inner_networks() const {
 
 event::ptr primitive_inst::execute(const std::vector<event::ptr>& events) {
     OV_ITT_SCOPED_TASK(ov::intel_gpu::itt::domains::intel_gpu_plugin, openvino::itt::handle("primitive_inst::execute: " + id()));
-    const auto primitive_id = id();
+    const auto& primitive_id = id();
```

Comment:
Could you also fix it in other places:
- https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/graph/primitive_inst.cpp#L1500
- https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gpu/src/graph/primitive_inst.cpp#L528

Reply:
done